config/lepao.yml

```diff
@@ -0,0 +1,31 @@
+# PURL configuration for http://purl.obolibrary.org/obo/lepao
+
+idspace: LEPAO
+base_url: /obo/lepao
+
+products:
+- lepao.owl: https://raw.githubusercontent.com/insect-morphology/lepao/master/lepao.owl
+- lepao.obo: https://raw.githubusercontent.com/insect-morphology/lepao/master/lepao.obo
+- lepao.json: https://raw.githubusercontent.com/insect-morphology/lepao/master/lepao.json
```

Comment:
There is no `lepao.json` file in the GitHub repository.

Reply:
You can either remove this line from the PURL config (easier), or create the file.

Reply:
eliminated lepao.json